src/Particles/babylon.particleSystem.ts

```diff
@@ -137,6 +140,106 @@
         }
         // end of sheet animation
 
+        // Emitter Type
+
+        /**
+         * This bool is used to specify whether to use the default sphere calculated direction or to calculate the direction using direction1 and direction2 properties.

```

Comment:
Hey it's me the picky guy :D
As we will probably add more in the future I would recommend to provide an interface like IParticleEmitterType which provide a startPositionFunction

And then we can provide 3 emitters (box, sphere and cone)

Also we can provide static on the ParticleSystem class like CreateBoxEmitter(size) or CreateSphereEmitter(radius)